transifex/native/cds.py

```diff
@@ -27,6 +27,19 @@
 }
 
 
+def EtagStore(object):
```

Comment:
I am rather fond of using python's `__g/setitem__` methods.

In fact, if we were doing YAGNI we could use `dict`s for `EtagStore` instances and when the time came to add more functionality, we could define a `EtagStore` class that would inherit from `collections.UserDict` and override the `__getitem__` and `__setitem__` methods.

AFAIK the rest of the company prefers enforcing this kind of structure early on so I am more than ok with the current implementation.